JustSaying.Sample.Restaurant.OrderingApi/Startup.cs

```diff
@@ -0,0 +1,109 @@
+using Amazon;
+using JustSaying.Sample.Restaurant.Models;
+using JustSaying.Sample.Restaurant.OrderingApi.Handlers;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Swashbuckle.AspNetCore.Swagger;
+
+namespace JustSaying.Sample.Restaurant.OrderingApi
+{
+    public class Startup
+    {
+        private readonly string _awsRegion;
+        private readonly string _awsServiceUrl;
+
+        public Startup(IConfiguration configuration)
+        {
+            _awsRegion = configuration.GetValue<string>("AWSRegion");
+            _awsServiceUrl = configuration.GetValue<string>("AWSServiceUrl");
+        }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+
+            services.AddJustSaying(config =>
+            {
+                config.Client(x =>
+                {
+                    if (!string.IsNullOrWhiteSpace(_awsServiceUrl))
+                    {
+                        // The AWS client SDK allows specifying a custom HTTP endpoint.
+                        // For testing purposes it is useful to specify a value that
+                        // points to a docker image such as `p4tin/goaws` or `localstack/localstack`
+                        x.WithServiceUrl(_awsServiceUrl);
```

Comment:
Add a call to `WithAnonymousCredentials()` (see #528).

Reply:
done